app/views/layouts/admin.html.erb

```diff
@@ -37,22 +37,23 @@
     <div class="crayons-header__container">
       <%= render "layouts/logo" %>
 
-      <h1 class="pl-4 fs-l">
-        <a href="<%= admin_path %>">Admin</a>
-        <span class="color-base-60">&raquo;</span> <span class="fw-bold">
+      <h1 class="pl-4 fs-base hidden s:block">
```

Comment:
I know it's not really in scope for this project but I had to change it really badly.